dataprofiler/profilers/profiler_options.py

```diff
@@ -982,6 +982,130 @@ def _validate_helper(self, variable_path: str = "CorrelationOptions") -> list[st
         return errors
 
 
+class HyperLogLogOptions(BooleanOption):
+    """Options for alternative method of gathering unique row count."""
+
+    def __init__(
+        self, is_enabled: bool = False, seed: int = 0, register_count: int = 10
+    ) -> None:
+        """
+        Initialize options for the hyperloglog method of gathering unique row count.
+
+        :ivar is_enabled: boolean option to enable/disable.
+        :vartype is_enabled: bool
+        :ivar seed: seed used to set HLL hashing function
+        :vartype seed: int
+        :ivar register_count: number of registers is equal to 2^register_count
+        :vartype register_count: int
+        """
+        BooleanOption.__init__(self, is_enabled=is_enabled)
+        self.seed = seed
+        self.register_count = register_count
+
+    def _validate_helper(self, variable_path: str = "HyperLogLogOptions") -> list[str]:
+        """
+        Validate the options do not conflict and cause errors.
+
+        :param variable_path: current path to variable set.
+        :type variable_path: str
+        :return: list of errors (if raise_error is false)
+        :rtype: list(str)
+        """
+        errors = super()._validate_helper(variable_path=variable_path)
+
+        if not isinstance(self.register_count, int):
+            errors.append(f"{variable_path}.register_count must be an integer.")
+        elif self.register_count <= 0:
+            errors.append(f"{variable_path}.register_count must be greater than 0.")
+
+        if not isinstance(self.seed, int):
+            errors.append(f"{variable_path}.seed must be an integer.")
+        return errors
+
+
+class UniqueCountOptions(BooleanOption):
```

Comment:
curious if `RowStatisticsOptions`, `UniqueCountOptions`, or `HyperLogLogOptions` should inherit `BooleanOption` or `BaseInspectorOptions`

Reply:
@JGSweets

Reply:
I'm wondering it should be a `BaseInspectorOption` (e.g. `TextProfilerOptions` in `UnstructuredOptions`)